core/scripts/chaincli/handler/keeper.go

```diff
@@ -631,6 +632,16 @@ func (k *Keeper) deployUpkeeps(ctx context.Context, registryAddr common.Address,
 			if err != nil {
 				log.Fatal(i, upkeepAddr.Hex(), ": RegisterUpkeep failed - ", err)
 			}
+
+			// Start up log trigger cycle
+			logUpkeepStartTx, err := logUpkeepCounter.Start(k.buildTxOpts(ctx))
+			if err != nil {
+				log.Fatal("failed to start log upkeep counter", err)
```

Comment:
open to the idea of making this non-fatal since someone can re-issue the `start()` command directly from contract. either way...